docker/secondary-analysis-python/tools/file_utils/file_utils/utils.py

```diff
@@ -0,0 +1,28 @@
+import requests
+
+
+def get_file_by_uuid(file_id):
+    url = 'https://dss.dev.data.humancellatlas.org/v1/files/{}?replica=gcp'.format(file_id)
+    print('GET {0}'.format(url))
+    response = requests.get(url)
+    print('{0}'.format(response.status_code))
+    print('{0}'.format(response.text))
+    return response.json()
+
+
+def get_manifest_files(bundle_uuid, bundle_version):
+    url = "https://dss.dev.data.humancellatlas.org/v1/bundles/" + bundle_uuid + "?version=" + bundle_version + "&replica=gcp&directurls=true"
```

Comment:
should this version `v1` be a parameter (version=`v1`) -- do we expect them to change this? 
```
def get_manifest_files(bundle_uuid, bundle_version, api_version='v1'):
    url = ("https://dss.dev.data.humancellatlas.org/{api_version}/bundles/{uuid}?version="
           "{bundle_version}&replica=gcp&directurls=true".format(
               api_version=api_version, uuid=bundle_uuid, bundle_version=bundle_version))
```

Reply:
I'm not sure about the api versioning, but I can check in with @dshiga about that!

Reply:
Also now that I'm looking at this again I think the dss url should also be a parameter to easily switch between staging and dev.

Reply:
Great!

Reply:
@dshiga mentioned the api version won't change frequently, so I included that as part of the `dss_url` parameter (e.g. "https://dss.dev.data.humancellatlas.org/v1")